electron/src/holochain.ts

```diff
@@ -67,13 +67,13 @@ const devOptions: HolochainRunnerOptions = {
   dnaPath: profilesDnaPath, // preload
   datastorePath: process.env.ACORN_TEST_USER_2
     ? '../tmp/databases'
-    : '../tmp2/databases',
+    : path.join(__dirname, '../../tmp2/databases'),
   appId: MAIN_APP_ID,
   appWsPort: process.env.ACORN_TEST_USER_2 ? 8899 : 8888,
   adminWsPort: process.env.ACORN_TEST_USER_2 ? 1236 : 1234,
   keystorePath: process.env.ACORN_TEST_USER_2
     ? '../tmp/keystore'
-    : '../tmp2/keystore',
+    : path.join(__dirname, '../../tmp2/keystore'),
```

Comment:
I did the same steps outlined in #36 on my mac, and it seems to be working without having the directory paths update as seen here

Reply:
Ok, that's ok. They refer to the same folder. This way creates more clarity on which folder that is though, because it is specified in an absolute way, as opposed to a relative way. 

This wasn't the fix then.